incubator/stellar-core/templates/_helpers.tpl

```diff
@@ -36,5 +36,14 @@ Create a default fully qualified app name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 */}}
 {{- define "stellar-core.postgresql.fullname" -}}
-{{- printf "%s-%s" .Release.Name "postgresql" | trunc 63 | trimSuffix "-" -}}
+{{- if .Values.postgresql.fullnameOverride -}}
```

Comment:
Is this defined in the subchart?
If so probably it's better to use it there instead of maintaining it in two places.

Reply:
It is. However, using `{{ template "postgresql.fullname" .}}` uses the wrong values (e.g., `nameOverride` from the parent chart – I've seen this in other charts) and `{{ template "postgresql.fullname" .Values.postgresql }}` fails because it can't find certain values if I don't redefine them (e.g., `postgresql.fullnameOverride`). Is there a better way to use the subchart's template?